scripts/sed

```diff
@@ -26,7 +26,7 @@ s/([Ss])ummarise/\1ummarize/g
 s/([Ss])ynchronisation/\1ynchronization/g
 s/([Ss])ynchronised/\1ynchronized/g
 s/\<([Uu]tili)s/\1z/g
-s/([Pp])rogramm(e)?(s)?/\1rogram\3/g
+s/^([Pp])rogramm(e)?(s)?$/\1rogram\3/g
```

Comment:
Why only as single word in a line?

Reply:
Thanks for the finding, you are right, I mixed up `^` and `\<` resp. `$` and `\>`.
Fixed it now.